cluster/gce/gci/master-helper.sh

```diff
@@ -135,10 +137,11 @@ function create-master-instance-internal() {
   disk="${disk},boot=no"
   disk="${disk},auto-delete=no"
 
+  # shellcheck disable=SC2034  # Unused variables left for readability
```

Comment:
on modifying:
```
In ./cluster/gce/gci/master-helper.sh line 138:
  for attempt in $(seq 1 ${retries}); do
  ^-^ SC2034: attempt appears unused. Verify use (or export if used externally).
Ref: https://github.com/koalaman/shellcheck/wiki/SC2034
```
I don't really know if I need to export here, but I've chosen to disable it.

Reply:
If it fails you an add a line that says something like "attempt number $attempt failed, will retry again in $sleep_sec" or something